src/utils/formData.js

```diff
@@ -0,0 +1,20 @@
+const defaultKey = 'cms.form';
```

Comment:
@hunyoboy not sure a default makes sense here. not much to lose by passing in `news.article`